app/models/guide.rb

```diff
@@ -50,6 +50,15 @@ def has_published_edition?
     editions.where(state: "published").any?
   end
 
+  def has_unpublished_edition?
+    editions.where(state: "unpublished").any?
+  end
```

Comment:
Oh.. could they both be moved to private then? If we're using the methods anywhere outside the class they should be private. Right?